src/lsdb/dask/crossmatch_catalog_data.py

```diff
@@ -36,26 +37,33 @@ def perform_crossmatch(
     left_pix,
     right_pix,
     right_margin_pix,
+    aligned_pix,
```

Comment:
This is a terrifying number of positional arguments. Maybe we should switch to using optional keyword args?

But that shouldn't block this PR.